corehq/apps/hqwebapp/templates/hqwebapp/partials/global_navigation_bar.html

```diff
@@ -31,7 +32,13 @@
             {% else %}
                 <a class="btn btn-project{% if is_project_settings %} btn-project-active{% endif %}"
                    href="{% url "domain_settings_default" domain %}">
+                    {% if request|tzmigration_status == 'complete' %}
+                    <i class="icon-globe" title="New timezone behavior enabled"></i>
+                    {% elif request|tzmigration_status == 'in_progress' %}
+                    <i class="icon-globe" title="Timezone migration started but not complete"></i>
+                    {% else %}
                     <i class="icon-cog"></i>
+                    {% endif %}
```

Comment:
Hmm, well that's a good point, though I think it's already different (e.g. I don't see the cog icon that is in this file); and maybe more to the point, this is sort of a short-term thing to make it easy for me to quickly distinguish when looking at a domain whether it's been migrated. (Like when I'm debugging.) I'll probably change it and/or remove it eventually and I think right now bs3 isn't used anywhere except dashboard, so will be nice to not have to try and keep those in sync. Let me know if that's too lazy of me and I'll update bs3 template too.